schemas/rules/operators/comparison/less-than.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/rules/operators/comparison/less-than.json",
+  "description": "Evaluates less than, takes two arguments",
+  "type": "object",
+  "properties": {
+    "<": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "$ref": "https://eq.ons.gov.uk/rules/definitions.json#/any_value_or_operator"
```

Comment:
As discussed, we should look at restricting these to any value or non-boolean operators instead of all operators.

Reply:
Done in https://github.com/ONSdigital/eq-questionnaire-validator/pull/96/commits/8da0b1f405624c0861dd104460b9043e11ec9332